tests/test_tokenization_common.py

```diff
@@ -712,3 +733,147 @@ def _check_no_pad_token_padding(self, tokenizer, sequences):
 
             # add pad_token_id to pass subsequent tests
             tokenizer.add_special_tokens({"pad_token": "<PAD>"})
+
+    @require_torch
+    def test_torch_encode_plus_sent_to_model(self):
+        from transformers import MODEL_MAPPING, TOKENIZER_MAPPING
+
+        MODEL_TOKENIZER_MAPPING = merge_model_tokenizer_mappings(MODEL_MAPPING, TOKENIZER_MAPPING)
+
+        tokenizer = self.get_tokenizer()
+
+        if tokenizer.__class__ not in MODEL_TOKENIZER_MAPPING:
+            return
+
+        config_class, model_class = MODEL_TOKENIZER_MAPPING[tokenizer.__class__]
+        config = config_class()
+
+        if config.is_encoder_decoder or config.pad_token_id is None:
+            return
```

Comment:
It is necessary to return when the `pad_token_id` is `None` even when not creating batches, as the `FastTokenizer` will always call `batch_encode_plus` underneath, which doesn't care about the number of sequences in the batch (even if it is 1 as called by `encode_plus`, if the tokenizer doesn't have a padding ID it will raise a `ValueError`) cc @mfuntowicz